src/lib.rs

```diff
@@ -115,6 +126,10 @@ fn val_from_attrs(attrs: &[Attribute], attrname: &str) -> Option<String> {
     None
 }
 
+fn contains_in_attrs(attrs: &[Attribute], attrname: &str) -> bool {
```

Comment:
How about just `has_attr`?

Reply:
I imitated `val_from_attrs`, but that is a better idea.
https://github.com/adwhit/diesel-derive-enum/blob/36c44a2310674365915bd87734491227593a517d/src/lib.rs#L96-L116

I will change it to `has_attr`.